cmd/bucket-handlers.go

```diff
@@ -503,7 +503,7 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 			DeleteMarkerReplicationStatus: dObjects[i].DeleteMarkerReplicationStatus,
 			PurgeTransitioned:             dObjects[i].PurgeTransitioned,
 		}]
-		if errs[i] == nil || isErrObjectNotFound(errs[i]) || isErrVersionNotFound(errs[i]) {
+		if errs[i] == nil || isErrObjectNotFound(errs[i]) || isErrVersionNotFound(errs[i]) || isErrInvalidVersionID(errs[i]) {
```

Comment:
This would never happen because of uuid.Parse is never invoked invalid versions at underlying layer, underling layer always returns VersionNotFound

Reply:
that's true. removing this check